### PR TITLE
The Sidebar Journey

### PR DESCRIPTION
**Pull request by brett-yoncak**

### CODE REVIEW

File: src/components/HeaderBar.vue [hunk 1]
```diff
@@ -44,7 +45,7 @@ const props = defineProps({ allSports: { type: Array, required: true } })
   </div>
 </template>
 
-<style>
+<style sass>
 .container {
   width: 100%;
 }
```
Comment (ricardodelgado-penn) [inline src/components/HeaderBar.vue:51]: All style blocks should be `scoped`

**Review by ricardodelgado-penn — COMMENTED** [review]
Looks like the baseball png got back into the code?

File: src/App.vue [hunk 2]
```diff
@@ -7,7 +7,12 @@ import { RouterView } from 'vue-router';
...
 
 <style>
+html {
+  height: 100;
+}
+
 body {
+  height: 100;
   background-color: #101c33;
 }
```
Comment (ricardodelgado-penn) [inline src/App.vue:17]: Looks like you left off the values.

File: src/components/SideBar.vue [hunk 3]
```diff
@@ -15,3 +20,33 @@ let pages = ['Home', 'Sports', 'Events', 'Teams', 'Preferences' ]
     </div>
   </div>
 </template>
+
+<style scoped>
+.container {
+    display: flex;
+    flex-direction: column;
+}
```
Comment (ricardodelgado-penn) [inline src/components/SideBar.vue:28]: Also make sure all style blocks are `scss`

File: src/components/SideBar.vue [hunk 4]
```diff
@@ -15,3 +20,33 @@ let pages = ['Home', 'Sports', 'Events', 'Teams', 'Preferences' ]
...
+.page {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    background-color: #182948;
+    font-size: 16px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.pages-wrapper {
+    color: white;
```
Comment (ricardodelgado-penn) [inline src/components/SideBar.vue:46]: Space between lines.

File: src/views/HomeView.vue [hunk 5]
```diff
@@ -48,8 +50,42 @@ onMounted(() => {
...
+        :name="allSports.name"
+      />
+    </div>
+    <div class="sidebar">
+      <SideBar />
+    </div>
+    <div class="body">
+      body
+    </div>
+  </div>
```
Comment (ricardodelgado-penn) [inline src/views/HomeView.vue:66]: Use semantic HTML where you can. I.e. `<main>` `<aside>` `<header>` etc

File: src/views/HomeView.vue [hunk 6]
```diff
@@ -48,8 +50,42 @@ onMounted(() => {
...
+    grid-area: 🟪;
+}
+
+.sidebar {
+    grid-area: 🟨;
+}
+
+.body {
+    grid-area: 🟥;
+}
```
Comment (ricardodelgado-penn) [inline src/views/HomeView.vue:90]: Use words, not color blocks.

**Review by ricardodelgado-penn — COMMENTED** [review]
Looks like you're missing icons to go with the sidebar items. Also your padding is off, give them some breathing room, and also don't let there be a border around the sidebar, it should sit up against the side of the page. 

Screen Shot 2022-06-27 at 2 21 53 PM

It also looks like you aren't filling up the entire vertical space on the page.

File: src/components/SideBar.vue [hunk 7]
```diff
@@ -1,10 +1,15 @@
...
+
 let pages = ['Home', 'Sports', 'Events', 'Teams', 'Preferences' ]
 </script>
 
 <template>
   <div class="container">
     <div class="pages-wrapper">
+      <div class="logo-block">
+        <LogoBlock />
+      </div>
```
Comment (ricardodelgado-penn) [inline src/components/SideBar.vue:12]: In this section, let's make a smaller component to house the sidebar items.

Comment (brett-yoncak) [reply src/components/SideBar.vue:12]: just a little confused on this one. I'll ask about it tomorrow morning.

**Pull request merged**